dotrix_core/src/renderer/light/raw_point_light.rs

```diff
@@ -0,0 +1,31 @@
+use super::{ Color };
+use dotrix_math::{ Vec4 };
+
+/// Internal struct that is passed to shaders
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct RawPointLight {
```

Comment:
What about call it PointLightUniform? 
Uniform is a standard naming for data structures passed to shaders.

Reply:
I can rename if it is a standard naming :)

Reply:
Unity thing? :D